### PR TITLE
Feature/yarn setup

### PR DESCRIPTION
**Pull request by Mrozelek**

### CODE REVIEW

**Comment by Mrozelek** [comment]
We could also separate lint command into two commands:
lint - for eslint
lint:scss - for stylelint

File: package.json [hunk 1]
```diff
@@ -7,7 +7,7 @@
     "start": "webpack -w",
     "build": "webpack",
     "test": "jest",
-    "lint": "eslint . --ext .ts"
+    "lint": "eslint . --ext .ts && stylelint 'src/**/*.scss'"
```
Comment (Mrozelek) [inline package.json:10]: test

Comment (Mrozelek) [reply package.json:10]: It would go like this:
```suggestion
    "lint": "eslint . --ext .ts"
    "lint:scss": "stylelint 'src/**/*.scss'"
```

**Review by kami3la — APPROVED** [review]
For me, the current lint command is fine

**Comment by Mrozelek** [comment]
Project needs to be reconfigured in order to host on gh pages.

**Pull request closed without merging**